node_modules/game-iverse/platformer/platformer.html

```diff
@@ -0,0 +1,41 @@
+<head>
+    <script src="platformer.js"></script>
+    <script src="levels.js"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+
+
+<body>
+    <div id="insert"></div><div id="insert2">Coins: 0</div>
+    <button value="Refresh Page" onClick="window.location.reload();"> Restart </button> 
+  <script>
+    async function runGame(plans, Display) {
+    let lives = 3;
+    for (let level = 0; level < plans.length && lives > 0;) {
+      console.log(`Level ${level + 1}, lives: ${lives}`);
+      
+      document.getElementById("insert").innerHTML = 'Level ' + (level + 1) + ', lives: ' + lives;
+      let status = await runLevel(new Level(plans[level]),
+                                  Display);
+      if (status == "won") level++;
+      else lives--;
+    }
+    if (lives > 0) {
+      document.getElementById("insert").innerHTML = "YOU WON!!!";
+      document.getElementById("insert2").innerHTML = "";
+    } else {
+      document.getElementById("insert").innerHTML = "GAME OVER";
+      document.getElementById("insert2").innerHTML = "";
+
+    }
+  }
+  runGame(GAME_LEVELS, DOMDisplay);
+
+
+  </script>
+</br>
+</body>
+
+
+</body>

```

Comment:
Should place the `platformer/` folder in the root of the repo not in this `node_module/`